### DISCUSSION TITLE
How to catch double-tap event on iPad?

### DISCUSSION THREAD

**Discussion by IS-hanabusa** [fabricjs/fabric.js#8769] [Q&A]
I'm trying to get it to work on my iPad, how do I catch the double tap event?

**Comment by iocoker** [comment]
Have you tried `mouse:dblclick` on the canvas yet? I think that should capture the double tap event as well.

**Reply by IS-hanabusa** [reply]
Thanks for the response.
I tried it with just the mouse:dblclick event, but it didn't catch it.

**Reply by iocoker** [reply]
@IS-Hana Can you please create a [jsfiddle ](https://jsfiddle.net/) to show your code?

**Reply by Saqib92** [reply]
You should use Hammer.js for iPad (Touch Screen) events.
